### DISCUSSION TITLE
Combobox如何修改选中项颜色

### DISCUSSION THREAD

**Discussion by jsrgckf** [HandyOrg/HandyControl#1359] [Q&A]
主题色我改成了绿色的，但是Combobox的选中色是蓝色的，感觉不协调，所以问一下如何修改选中项颜色？

**Comment by CodingOctocat** [comment]
`Combobox` 选中的背景颜色使用的是  `{DynamicResource PrimaryBrush}`，而 PrimaryBrush 又使用了 `<Color x:Key="PrimaryColor">#326CF3</Color>` 和 `<Color x:Key="DarkPrimaryColor">#326CF3</Color>`，如果你的主题只有一个，那么你可以在自己的资源也定义同名的 `PrimaryColor` 和 `DarkPrimaryColor` 为绿色，这样你所有的首选色都变成了绿色。
